### PR TITLE
Improve existing codebase

### PR DESCRIPTION
**Pull request by dma1979**
All changes are synchronized with existing auto-formatting in Intellij Idea(based on current settings) and allow to have consistent codebase

### CODE REVIEW

File: src/main/java/one/util/streamex/TreeSpliterator.java [hunk 1]
```diff
@@ -122,34 +123,34 @@ public void run() {
...
                 }
             }
-            if(t instanceof RuntimeException)
-                throw (RuntimeException)t;
-            if(t instanceof Error)
-                throw (Error)t;
+            if (t instanceof RuntimeException)
+                throw (RuntimeException) t;
+            if (t instanceof Error)
+                throw (Error ) t;
```
Comment (amaembo) [inline src/main/java/one/util/streamex/TreeSpliterator.java:153]: It looks like the whitespace before closing parenthesis is redundant.

File: src/test/java/one/util/streamex/BaseStreamExTest.java [hunk 2]
```diff
@@ -59,9 +59,8 @@ public void testStreamConsumed() {
     @Test
     public void testClose() {
         List<String> closeHandlers = new ArrayList<>();
-        StreamEx<Integer> stream = StreamEx.of(Stream.of(1,2,3).onClose(() -> closeHandlers.add("Orig stream")))
-            .onClose(() -> closeHandlers.add("StreamEx"))
-            .map(x -> x*2)
+        StreamEx<Integer> stream = StreamEx.of(Stream.of(1, 2, 3).onClose(() -> closeHandlers.add("Orig stream")))
+            .onClose(() -> closeHandlers.add("StreamEx")).map(x -> x * 2)
```
Comment (amaembo) [inline src/test/java/one/util/streamex/BaseStreamExTest.java:63]: It looks like removing the linebreak before .map is unnecessary here. WDYT?

File: src/test/java/one/util/streamex/StreamExHeadTailTest.java [hunk 3]
```diff
@@ -300,7 +300,8 @@
     private static <T> StreamEx<T> twoFilters(StreamEx<T> input, Predicate<T> f1, Predicate<T> f2, Stream.Builder<T> buf) {
         return input.headTail((head, tail) -> {
             StreamEx<T> res = twoFilters(tail, f1, f2, buf);
-            if(f2.test(head)) buf.add(head);
+            if (f2.test(head))
+                buf.add(head);
```
Comment (amaembo) [inline src/test/java/one/util/streamex/StreamExHeadTailTest.java:304]: If linebreak is added, braces should be added as well.

**Comment by dma1979** [comment]
Code review remarks fixed

**Pull request merged**